### PR TITLE
releasenotes for depp 2022-12-14

### PR DESCRIPTION
**Pull request by faselm**

### CODE REVIEW

File: chsdi/static/doc/source/releasenotes/index.rst [hunk 1]
```diff
@@ -14,6 +14,54 @@ Release Notes
...
+`API <//api3.geo.admin.ch>`__
+'''''''''''''''''''''''''''''
+
+`MAP <//map.geo.admin.ch>`__
+'''''''''''''''''''''''''''''
+
+Geodata
+*******
++--------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| New    | `Building zones Switzerland (harmonized) <//map.geo.admin.ch/?layers=ch.are.bauzonen>`__ (ch.are.bauzonen)                                                                                                        |
```
Comment (rebert) [inline chsdi/static/doc/source/releasenotes/index.rst:34]: is this a new layer?

File: chsdi/static/doc/source/releasenotes/index.rst [hunk 2]
```diff
@@ -14,6 +14,54 @@ Release Notes
...
++--------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| New    | `Restrictions for drones <//map.geo.admin.ch/?layers=ch.bazl.einschraenkungen-drohnen>`__ (ch.bazl.einschraenkungen-drohnen)                                                                                      |
```
Comment (rebert) [inline chsdi/static/doc/source/releasenotes/index.rst:36]: is this a new layer?

File: chsdi/static/doc/source/releasenotes/index.rst (hunk 1 above)
Comment (mariokeusen) [reply chsdi/static/doc/source/releasenotes/index.rst:34]: No

File: chsdi/static/doc/source/releasenotes/index.rst (hunk 2 above)
Comment (mariokeusen) [reply chsdi/static/doc/source/releasenotes/index.rst:36]: No, we'll have a large model change on this by Dec 14th

Comment (mariokeusen) [reply chsdi/static/doc/source/releasenotes/index.rst]: Moved those 2 to the Updates...

**Review by consta-v — APPROVED** [review]

**Review by ltclm — APPROVED** [review]
thank you

**Comment by mariokeusen** [comment]
@davidoesch : right, I have claimed the texts from PN as foreseen in our decomm process.

**Pull request merged**